frontend/ui/data/css_tweaks.lua

```diff
@@ -191,40 +191,72 @@ h1, h2, h3, h4, h5, h6 { hyphens: none !important; }
     {
         title = _("Miscellaneous"),
         {
-            id = "table_row_odd_even";
-            title = _("Alternate background color of table rows"),
-            css = [[
+            title = _("Tables"),
+            {
+                id = "table_full_width";
+                title = _("Make tables use the full width"),
+                description = _("Have table use the full width of the page. (Tables with small content now uses only the width needed to display that content. This restores the previous behaviour.)"),
```

Comment:
```suggestion
                description = _("Make table expand to the full width of the page. (Tables with small content now use only the width needed to display that content. This restores the previous behavior.)"),
```

(Regarding behavior, I think we were going for a more AmE speling on the whole.)